### PR TITLE
cloud: release notes add 0720

### PR DESCRIPTION
**Pull request by Oreoxmt**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

cloud: release notes add 0720

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by hfxsd — APPROVED** [review]

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 1]
```diff
@@ -8,6 +8,23 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## July 20, 2023
+
+**General changes**
+
+- Data Service (beta) supports pagination for `GET` endpoints.
+
+    This feature improves the efficiency of handling large datasets and enhances the development experience.
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:17]:
```suggestion
- Data Service (beta) supports pagination for `GET` requests.

    This feature improves the efficiency of handling large datasets and enhances the development experience.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 2]
```diff
@@ -8,6 +8,23 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
...
+
+    For `GET` requests, you can paginate results by specifying `api_page` and `api_page_size` as query parameters when calling the endpoint. For example, to get the second page with 10 items per page, you can use the following command:
+
+    ```bash
+    curl --digest --user '<Public Key>:<Private Key>' \
+      --request GET 'https://<region>.data.tidbcloud.com/api/v1beta/app/<App ID>/endpoint/<Endpoint Path>?api_page=2&api_page_size=10'
+    ```
+
+    For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).
```
Comment (Oreoxmt) [inline tidb-cloud/tidb-cloud-release-notes.md:26]:
```suggestion
    Note that this feature is available only for `GET` requests where the last query is a `SELECT` statement.

    For more information, see [Call an endpoint](/tidb-cloud/data-service-manage-endpoint.md#call-an-endpoint).
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 3]
```diff
@@ -8,6 +8,23 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## July 20, 2023
+
+**General changes**
+
+- Data Service (beta) supports pagination for `GET` endpoints.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md]:
```suggestion
- [Data Service (beta)]([Data Service (beta)](https://tidbcloud.com/console/data-service)) supports pagination for `GET` endpoints.
```

File: tidb-cloud/tidb-cloud-release-notes.md [hunk 4]
```diff
@@ -8,6 +8,25 @@ aliases: ['/tidbcloud/supported-tidb-versions','/tidbcloud/release-notes']
 
 This page lists the release notes of [TiDB Cloud](https://www.pingcap.com/tidb-cloud/) in 2023.
 
+## July 20, 2023
+
+**General changes**
+
+- Data Service (beta) supports pagination for `GET` requests.
```
Comment (qiancai) [inline tidb-cloud/tidb-cloud-release-notes.md:15]:
```suggestion
- [Data Service (beta)](https://tidbcloud.com/console/data-service) supports pagination for `GET` requests.
```

**Review by qiancai — APPROVED** [review]
Rest LGTM

**Comment by Oreoxmt** [comment]
The release note has been included and can be found in the the feature document: https://github.com/pingcap/docs/pull/14237/commits/294b884700cf5e44c4677a109f2a7e187c787047

**Pull request closed without merging**